### DISCUSSION TITLE
Notification email about new comment

### DISCUSSION THREAD

**Discussion by falkensteg-gmbh** [outline/outline#7934] [Ideas]
the notification email about a new comment should include the comment itself in the text, so that the receipient does not have to log in into outline to read the comment.

**Comment by tommoor** [comment]
They do and have for a while, sounds like you need to update your instance ;)

**Reply by falkensteg-gmbh** [reply]
ooopsi, sorry for that.
btw, it is a great piece of software. thanks a lot for your amazing work!
